obs/window-basic-main.cpp

```diff
@@ -80,6 +80,11 @@ OBSBasic::OBSBasic(QWidget *parent)
 {
 	ui->setupUi(this);
 
+	QString styleSheetPath(os_get_config_path("obs-studio/stylesheet.qss"));
```

Comment:
Note that os_get_config_path currently doesn't return a constant character pointer, it returns an allocation.  The BPtr<> template class should be used on this allocation to automatically free it for C++ (in this case BPtr<char>).

This isn't entirely your fault however, I probably should fix this function so that it does not return an allocation (despite it being convenient), but instead have it take in a character array and max size as arguments, as per normal C functions.

I may just go back and fix this throughout the project before merging the commit.